config/global.go

```diff
@@ -37,6 +39,15 @@ type GlobalConfig struct {
 	DaemonThreadsNum int
 	CacheGCPeriod    time.Duration
 	MirrorsConfig    MirrorsConfig
+	PrefetchFiles    PrefetchMessage
```

Comment:
This is not the right place for storing `prefetch information`. The struct `GlobalConfig` is only for storing configuration items from CLI and toml configuration file